src/tasks/AotCompilerTask/MonoAOTCompiler.props

```diff
@@ -17,7 +17,7 @@
     <MonoAOTCompilerDefaultAotArguments Condition="'$(TargetArchitecture)' == 'arm'" Include="mtriple=armv7-linux-gnueabi-thumbv7s" />
     <MonoAOTCompilerDefaultAotArguments Include="static" />
     <!-- Default trampolines run out for libraries tests -->
-    <MonoAOTCompilerDefaultAotArguments Include="nimt-trampolines=2000" />
+    <MonoAOTCompilerDefaultAotArguments Include="nimt-trampolines=4000" />
```

Comment:
https://github.com/dotnet/runtime/issues/53033#issuecomment-856973365

Reply:
Not sure about the correctness of this number but it seems that increasing helped to avoid the error of trampoline limitation.

Reply:
An example of the error: `((null) error) Ran out of trampolines of type 2 in '/data/user/0/net.dot.Microsoft.VisualBasic.Core.Tests/files/System.Private.CoreLib.dll' (limit 2000)`

Reply:
It still fails even on increased number, for ex.:
```
((null) error) Ran out of trampolines of type 3 in '/data/user/0/net.dot.System.Net.Sockets.Tests/files/System.Private.CoreLib.dll' (limit 4400)
```

Reply:
oh, just noticed that it's of `type 3`